tests/model_registry/conftest.py

```diff
@@ -0,0 +1,307 @@
+                        "imagePullPolicy": "IfNotPresent",
+                        "livenessProbe": {
+                            "exec": {
+                                "command": [
+                                    "/bin/bash",
+                                    "-c",
+                                    "mysqladmin -u${MYSQL_USER} -p${MYSQL_ROOT_PASSWORD} ping",
+                                ]
+                            },
+                            "initialDelaySeconds": 15,
+                            "periodSeconds": 10,
+                            "timeoutSeconds": 5,
+                        },
+                        "name": "mysql",
+                        "ports": [{"containerPort": 3306, "protocol": "TCP"}],
+                        "readinessProbe": {
+                            "exec": {
+                                "command": [
+                                    "/bin/bash",
+                                    "-c",
+                                    'mysql -D ${MYSQL_DATABASE} -u${MYSQL_USER} -p${MYSQL_ROOT_PASSWORD} -e "SELECT 1"',
+                                ]
+                            },
+                            "initialDelaySeconds": 10,
+                            "timeoutSeconds": 5,
+                        },
+                        "securityContext": {"capabilities": {}, "privileged": False},
+                        "terminationMessagePath": "/dev/termination-log",
+                        "volumeMounts": [
+                            {
+                                "mountPath": "/var/lib/mysql",
+                                "name": f"{DB_RESOURCES_NAME}-data",
+                            }
+                        ],
+                    }
+                ],
+                "dnsPolicy": "ClusterFirst",
+                "restartPolicy": "Always",
+                "volumes": [
+                    {
+                        "name": f"{DB_RESOURCES_NAME}-data",
+                        "persistentVolumeClaim": {"claimName": DB_RESOURCES_NAME},
+                    }
+                ],
+            },
+        },
+        wait_for_resource=True,
+    ) as mr_db_deployment:
+        mr_db_deployment.wait_for_replicas(deployed=True)
+        yield mr_db_deployment
+
+
+@pytest.fixture(scope="class")
+def model_registry_instance(
+    admin_client: DynamicClient,
+    model_registry_namespace: Namespace,
+    model_registry_db_deployment: Deployment,
+    model_registry_db_secret: Secret,
+    model_registry_db_service: Service,
+) -> Generator[ModelRegistry, Any, Any]:
+    with ModelRegistry(
+        name="model-registry",
+        namespace=model_registry_namespace.name,
+        label={
+            "app.kubernetes.io/name": "model-registry",
+            "app.kubernetes.io/instance": "model-registry",
+            "app.kubernetes.io/part-of": "model-registry-operator",
+            "app.kubernetes.io/managed-by": "kustomize",
```

Comment:
I think we can removed kustomize . I don't think it is mandatory, as i understand when we use kustomize tool people tend to add this label like fyi . Is it mandatory?